Source/Common/EnumHelper.cs

```diff
@@ -30,7 +30,9 @@ internal static class EnumHelper
                 result = Enum.Parse(enumType, value);
                 return true;
             }
+#pragma warning disable CA1031 // Do not catch general exception types
             catch
+#pragma warning restore CA1031 // Do not catch general exception types
```

Comment:
Is it worth moving this pragma (and the one below it about the expression body) to be for the whole file? Might be a little less noisy?